sound/soc/intel/common/soc-acpi-intel-rpl-match.c

```diff
@@ -347,6 +371,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_rpl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-rpl-rt711-l0-rt1318-l12-rt714-l3.tplg",
 	},
+	{
+		.link_mask = 0xF, /* 4 active links required */
+		.links = rpl_sdw_rt711_link0_rt1316_link12_rt714_link3,
+		.drv_name = "sof_sdw",
+		.sof_tplg_filename = "sof-rpl-rt711-l0-rt1316-l12-rt714-l3.tplg",
+	},
```

Comment:
@bardliao can I ask which device needs this? This looks rather similar to what I added for the Dell SKU 0BDA in https://github.com/thesofproject/linux/pull/4381/commits/8a472fbddc1335a95353f39ff64393301146705a

Reply:
I merged the code already btw @bardliao